static/css/components/editions.less

```diff
@@ -238,3 +248,13 @@ tbody {
 .editions-table__links {
   text-align: center;
 }
+
+a.paginate_button.current {
+  background-color: @lightest-grey;
+  border: 1px solid @lighter-grey;
+  color: @black;
+}
+
+#editions_paginate a.paginate_button {
+  border: none;
```

Comment:
Ah, this is overwriting the `border` above! I assume this was to override the default styling; let's remove the line causing trouble from `legacy-datatables`. That should fix it!